### ISSUE TITLE
Python 3.6 incompatibility because of enum34

### ISSUE THREAD

**Issue by twiecki** [conda-forge/pymc3-feedstock#10]
```
UnsatisfiableError: The following specifications were found to be in conflict:
  - pymc3 -> enum34 -> python 2.7*
  - python 3.6*
Use "conda info <package>" to see the dependencies for each package.
```

**Comment by ocefpaf** [comment]
My understating is that  `enum34` should be skipped for `skip: true  # [py>=34]` because it is built-in in those versions. From its description: `enum34 is the new Python stdlib enum module available in Python 3.4 backported for previous versions of Python from 2.4 to 3.3. tested on 2.6, 2.7, and 3.3+`

**Comment by ericmjl** [comment]
I think it's all solved! Closing issue.
